src/meshapi/models.py

```diff
@@ -165,31 +327,58 @@ class SectorStatus(models.TextChoices):
         ACTIVE = "Active"
         POTENTIAL = "Potential"
 
-    building = models.ForeignKey(Building, on_delete=models.PROTECT, related_name="sectors")
-    name = models.TextField()
+    building = models.ForeignKey(
+        Building,
+        on_delete=models.PROTECT,
+        related_name="sectors",
+        help_text="The building this sector is installed on",
+    )
+    name = models.TextField(
+        help_text="The name of the hub sector is installed on",
+    )
 
     radius = models.FloatField(
+        help_text="The radius to display this sector on the map (scale is km?)",
```

Comment:
What do you mean, "scale is km?" Do we not know what unit this radius is in? We should figure that out (out of scope of this PR, sure, but might wanna make an issue for it).

Reply:
I checked the existing map, confirmed this is `km`, I will fix this note